docs/devTools/README.md

```diff
@@ -0,0 +1,62 @@
+# Dev tools
+
+Local development tools for deploying the [any.sender contracts](https://github.com/PISAresearch/contracts.any.sender) and hosting a mock any.sender instance. We also have a live ropsten API for when you're ready to take it to the next level :)
+
+## Deploying contracts locally
+
+To deploy the contracts import the `deploy` object from the `@any-sender/client` package.  The deploy object has an `contracts` function which accepts a provider and returns a promise of the contracts. Call this at some point in your tests before wrapping objects with `any.sender()`.
+
+```
+contracts(provider: Provider, options?: DeployOptions) => Promise<Contracts>
+```
+
+#### provider: JsonRpcProvider | AsyncSendable | string
+Can be an ethers `JsonRpcProvider`, `AsyncSendable` or a json rpc url as a `string`.
+
+#### options?: DeployOptions
+```
+{
+    funder?: string | number;
+    shardInterval?: number;
+    compensationPeriod?: number;
+    withdrawalPeriod?: number;
+    deployMetaTxContracts?: boolean;
+}
+```
+
+* `funder`: The address or index of the wallet that will be used to fund the any.sender deployment. If not supplied last wallet returned by provider.listAccounts will be used for this.
```

Comment:
How does it work with an address? Should it not be a wallet instance? 

"If not supplied, then the last wallet ...."

Reply:
No, only has to be an address or an index.

Updated typo.